jenkins/ci.suse.de/pipelines/openstack-ardana.Jenkinsfile

```diff
@@ -248,7 +248,12 @@ pipeline {
           // packages. Can be removed when jenkins feature (JENKINS-48315) is merged.
           env.stage_after_update = "true"
 
-          cloud_lib.ansible_playbook('ardana-update', "-e cloudsource=$update_to_cloudsource")
+          if (update_to_cloudsource == '') {
+            cloud_lib.ansible_playbook('ardana-update', "-e cloudsource=$cloudsource")
```

Comment:
I probably should just remove the -e parameter item entirely since we are setting cloudsource to the same value as it already has...